### PR TITLE
Use itemRgb field for coloring bedlike tracks

### PR DESCRIPTION
**Pull request by pkerpedjiev**
## Description

> What was changed in this pull request?

This PR enables bedlike tracks to use the `itemRgb` field to color annotations.

See https://genome.ucsc.edu/FAQ/FAQformat.html#format1 for a description of the BED format as well as the location of the `itemRgb` field within it.

> Why is it necessary?

To follow existing standards

## Screenshot

image

Fixes #___

## Checklist

- [x] Set proper GitHub labels (e.g. v1.6+, ignore if you don't know)
- [x] Unit tests added or updated
- [x] Documentation added or updated
- [o] Example(s) added or updated
- [o] Update schema.json if there are changes to the viewconf JSON structure format
- [x] Screenshot for visual changes (e.g. new tracks or UI changes)
- [x] Updated CHANGELOG.md

### CODE REVIEW

File: docs/track_types.rst [hunk 1]
```diff
@@ -46,7 +46,8 @@ column" section).
 Intervals can visually encode information using the following three ``options``:
 
 - **colorEncoding**: int or null
```
Comment (alexander-veit) [inline docs/track_types.rst:48]: int, 'itemRgb' or null

Comment (alexander-veit) [reply docs/track_types.rst:48]: Also, update the `valueColumn` description to:
```
**valueColumn**: int or null
    Use one of the bed columns (1-based) to scale the annotations along the y axis. (default `null`)
```
`none` does not work.

**Comment by alexander-veit** [comment]
I did some tests with `valueColumn` and found a minor issue:
Screen Shot 2020-04-22 at 10 24 33 AM
The rectangles slightly overlap, which does not look great.

**Comment by pkerpedjiev** [comment]
Thanks Alex!

> I did some tests with valueColumn and found a minor issue:

This is actually expected. The annotations have a set height. There's an option to set their height to "scaled" which will scale it according to how many are stacked. Might be good to automatically switch to that option when switching to value scaling. I'll leave that for another PR though.

File: docs/track_types.rst (hunk 1 above)
Comment (pkerpedjiev) [reply docs/track_types.rst]: 👍 Good catch!

**Comment by alexander-veit** [comment]
Thanks. I am not sure if we are talking about the same thing though. When I look at the last yellow rectangle, I would have expected it to be a few pixels more to the right, so that it does not overlap with the blue one (wrt `x`-direction). Is this what you meant?

**Comment by pkerpedjiev** [comment]
Oh, indeed we are not talking about the same thing. I added an issue for the thing that you observed: https://github.com/higlass/higlass/issues/901

**Pull request merged**